bindings/rust/wasmedge-sys/src/instance/function.rs

```diff
@@ -321,6 +321,30 @@ impl Function {
         }
     }
 
+    pub fn create_async<T>(
```

Comment:
Please add doc comments for this new API. As for the format of the API doc comments, please refer to the existed. Thanks!